x-pack/plugins/security_solution/common/detection_engine/get_query_filter.ts

```diff
@@ -16,7 +16,7 @@ import {
   CreateExceptionListItemSchema,
 } from '../../../lists/common/schemas';
 import { ESBoolQuery } from '../typed_json';
-import { buildExceptionFilter } from './build_exceptions_filter';
+import { buildExceptionFilter } from '../shared_imports';
```

Comment:
I wonder if we should make more things available through the plugin so for example here we'd use the `lists` client and do something like `lists.queryBuilder`. Just a thought, nothing to do in this PR.